features/step_definitions/common_steps.rb

```diff
@@ -543,9 +544,12 @@
   else
     target_window = current_window
   end
-
-  within_window(target_window) do
-    disposition_parts = page.response_headers['Content-Disposition'].split(";")
+  if is_chrome
+    disposition_parts = inline_http_requests.map { |r| r.dig('content-disposition') }.compact[0].split(";")
+  else
+    within_window(target_window) do
+      disposition_parts = page.response_headers['Content-Disposition'].split(";")
+    end
```

Comment:
Why are we ending the `within_window` here?

Reply:
fixed in 7e1157b4134b0f73e55a4924d56638b43029e830